frontend/packages/console-app/locales/en/console-app.json

```
@@ -178,6 +178,15 @@
   "Namespace where the API configuration secret is located": "Namespace where the API configuration secret is located",
   "Scheduling disabled": "Scheduling disabled",
   "Approval required": "Approval required",
+  "Display Name": "Display Name",
+  "Status": "Status",
+  "The state of the plugin.": "The state of the plugin.",
+  "Enabled": "Enabled",
+  "A boolean to determine whether or not the plugin renders in console.": "A boolean to determine whether or not the plugin renders in console.",
```

What about shortening it? I think the "whether or not" is implied.

```suggestion
  "A boolean to determine if the plugin renders in console.": "A boolean to determine if the plugin renders in console.",
```

---

Sorry, @opayne1.  I rewrote this while making some additions and just noticed your comment.  The text now reads, `Whether or not the plugin renders in console.`. WDYT?